adapter/src/main/java/org/jvalue/ods/adapterservice/adapter/amqp/AmqpReceiver.java

```diff
@@ -0,0 +1,24 @@
+package org.jvalue.ods.adapterservice.adapter.amqp;
```

Comment:
I suppose you add that for communication between scheduler and adapter to trigger imports? 
If there is no reason why this absolutely needs to be included, please leave that out for now (since it is independent of the other changes) and do it later in another pull request.